### DISCUSSION TITLE
Welcome to paper-cred Discussions!

### DISCUSSION THREAD

**Discussion by vitorpamplona** [Path-Check/paper-cred#27] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DaevMithran** [comment]
If anyone get's a snapshot of my QR code then they can impersonate me. I believe binding credential is important, only an authenticated person should prove ownership of the QR!

**Reply by vitorpamplona** [reply]
Agree! That's why all Passes and Immunization records are indeed bound to a person. 

But let's not forget that payloads without any user binding exist and are used widely today (e.g. money bills, purchase receipts, etc.)

Not every use case needs binding.

**Reply by DaevMithran** [reply]
Thanks, I Agree. you mean the binding in passes and immunization records are the attributes specific to a holder like DOB, Name, Mobile number, etc. right?

**Reply by vitorpamplona** [reply]
Correct. That's more than enough to prove identity in COVID use cases (by showing the QR together with a Gov-issued ID).

**Reply by DaevMithran** [reply]
Can you give more information on how selective disclosure can be achieved in this model, which is mentioned as one of the benefits in the paper-cred specification

**Reply by vitorpamplona** [reply]
Sure thing! The idea is to provide 2-3 QRs with different information. You can see one idea here: https://vaccine-docs.pathcheck.org/specification/cdc-folding-card  with a fully functional demo here: https://github.pathcheck.org/index.v5.html

**Reply by vitorpamplona** [reply]
In that example, selective disclosure is a physical action of showing the appropriate QR and hiding the others by folding the card.

**Reply by Sodiaq** [reply]
> If anyone get's a snapshot of my QR code then they can impersonate me. I believe binding credential is important, only an authenticated person should prove ownership of the QR!
